src/routes.tsx

```diff
@@ -105,6 +106,7 @@ export function AppRoutes() {
       </Route>
       {/* Regions routes */}
       <Route path={`${basePath}/regions`} element={<RegionalRankedPage />} />
+      <Route path={`${basePath}/regions/:id`} element={<RegionDetailPage />} />
```

Comment:
If you ever want a page only visible in stage you can use something like this btw 🙂  

```
<Route element={<StagingProtectedRoute />}>
        <Route path={`${basePath}/regions`} element={<RegionalRankedPage />} />
        <Route
          path={`${basePath}/regions/:id`}
          element={<RegionDetailPage />}
        />
      </Route>
```